varianteval/scripts/README.md

```diff
@@ -96,4 +96,18 @@ sh survivor_merge.sh annotation_test/test_panel_survivor.txt {path to SURVIVOR e
 ```
 This command will generate a `merged.vcf` file containing the records representing the union of the records in the input vcfs
 specified in `test_panel_survivor.txt`, and it will also generate the following upsetplot:
-![](annotation_test/upset_survivor.png)
\ No newline at end of file
+![](annotation_test/upset_survivor.png)
+
+### build_igv_batch_script.py
+Procedure to generate an IGV batch script that can be used for automated screenshotting of different loci. There are three
```

Comment:
for generalizability, let's keep the igv batch script agnostic to survivor -- if filtering by support vector is desired, it can be done as a preprocessing step; we can also later consider supporting the support vector as an additional filter (to size and type) and as a field to split the folders by -- as commented below